src/Manager.php

```diff
@@ -49,8 +51,14 @@ public function __construct(
 
     public function userHasPermission($userId, string $permissionName, array $parameters = []): bool
     {
+        if ($userId === null && $this->guestRole !== null) {
```

Comment:
In case when both user ID and guest role is null in this method should return false instead of throws exception.